Resources/views/Crud/_boxes/show.html.twig

```diff
@@ -55,6 +55,7 @@
                                     {% endif %}
                                     <div class="row">
                                         <div class="col-lg-12">
+{#                                            {{ dump(content) }}#}
```

Comment:
@rugbymauri 🧐

Reply:
@rugbymauri Doesn't seem to have worked - please just remove the change completely, not add new commits to revert it.